nixos/tests/ceph.nix

```diff
@@ -51,6 +51,9 @@ import ./make-test.nix ({pkgs, ...}: rec {
         enable = true;
         daemons = [ "0" "1" ];
       };
+
+      # So that we don't have to battle systemd when bootstraping
+      systemd.targets.ceph.wantedBy = lib.mkForce [];
```

Comment:
Does this mean bootstrapping a new cluster is tricky?

Reply:
Both yes and no. The main reason I force the target off in the test is to not get a bunch of errors related to daemons not being able to find their files, because they haven't been bootstrapped yet, to clutter the test output. The errors are harmless but made it significantly harder to find actual errors in the output.